packages/syft/src/syft/oblv/model.py

```diff
@@ -0,0 +1,441 @@
+# stdlib
+from datetime import datetime
+import json
+import os
+from signal import SIGTERM
+import subprocess
+import sys
+import time
+from typing import Any
+from typing import List
+
+# third party
+from oblv import OblvClient
+import requests
+
+# relative
+from ..core.node.common.exceptions import OblvEnclaveError
+from ..core.node.common.exceptions import OblvUnAuthorizedError
+from ..util import bcolors
+from .oblv_proxy import check_oblv_proxy_installation_status
+
+
+class DeploymentClient:
+
+    # __attr_allowlist__ = ["deployment_id", "user_key_name", "client", "oblv_client"]
+
+    deployment_id: str
+    user_key_name: str
+    client: List[Any] = []  # List of domain client objects
+    oblv_client: OblvClient = None
+    __conn_string: str
+    __logs: Any
+    __process: Any
+
+    def __init__(
+        self,
+        domain_clients: List[Any],
+        deployment_id: str = None,
+        oblv_client: OblvClient = None,
+        user_key_name: str = "",
+    ):
+        self.deployment_id = deployment_id
+        self.user_key_name = user_key_name
+        self.oblv_client = oblv_client
+        self.client = domain_clients
+        self.__conn_string = ""
+        self.__process = None
+        self.__logs = None
+
+    def set_conn_string(self, URL):
+        self.__conn_string = URL
+
+    def initiate_connection(self, connection_port: int = 3032):
+        if check_oblv_proxy_installation_status() is None:
+            return
+        self.close_connection()  # To close any existing connections
+        public_file_name = os.path.join(
```

Comment:
MAYBE: Add subsequent ticket to move to `pathlib` from `os.path`

Reply:
Yep, will create a separate task for the same 👍